scipy/spatial/spherical_voronoi.py

```diff
@@ -0,0 +1,342 @@
+"""
```

Comment:
The file should be renamed to `_spherical_voronoi.py`, only the SphericalVoronoi class is public.